src/icrc2-minter/src/canister.rs

```diff
@@ -388,6 +389,25 @@ impl MinterCanister {
     pub fn idl() -> Idl {
         generate_idl!()
     }
+
+    /// Get mint orders for the given wallet address and token;
+    /// if `offset` and `count` are provided, returns a page of mint orders.
+    fn token_mint_orders(
+        wallet_address: H160,
+        src_token: Id256,
+        offset: Option<usize>,
+        count: Option<usize>,
+    ) -> Vec<(u32, SignedMintOrder)> {
+        get_operations_store()
+            .get_for_address(&wallet_address, offset, count)
+            .into_iter()
+            .filter_map(|(operation_id, status)| {
```

Comment:
This filter is applied after the offset and count, consequently the offset of the returned result and the number of returned entries do not match what requested